decidim-proposals/app/events/decidim/proposals/proposal_endorsed_event.rb

```diff
@@ -18,7 +18,11 @@ def endorser_path
       private
 
       def endorser
-        @endorser ||= Decidim::UserPresenter.new(extra[:endorser])
+        @endorser ||= Decidim::UserPresenter.new(endorser_user)
+      end
+
+      def endorser_user
+        @endorser_user ||= Decidim::User.find_by(id: extra[:endorser_id])
```

Comment:
Why not just `find`?

Reply:
`find_by` returns `nil` if not found, `find` raises an exception 🤷‍♂️ 

Don't have a strong opinion on this.